plugins/woocommerce/src/Blocks/BlockTypes/ProductCollection.php

```diff
@@ -445,8 +458,8 @@ public function add_navigation_link_directives( $block_content, $block, $instanc
 		$query_context                  = $instance->context['query'] ?? array();
 		$is_product_collection_block    = $query_context['isProductCollectionBlock'] ?? false;
 		$query_id                       = $instance->context['queryId'] ?? null;
-		$parsed_query_id                = $this->parsed_block['attrs']['queryId'] ?? null;
-		$is_enhanced_pagination_enabled = ! ( $this->parsed_block['attrs']['forcePageReload'] ?? false );
+		$parsed_query_id                = $$block->attributes['queryId'] ?? null;
+		$is_enhanced_pagination_enabled = ! ( $$block->attributes['forcePageReload'] ?? false );
```

Comment:
There's `$$` instead of single `$`. Is that on purpose? It throws a warning on the frontend: `Array to string conversion`:

<img width="1269" alt="Screenshot 2024-11-05 at 14 02 20" src="https://github.com/user-attachments/assets/5adb55b9-3f49-4b1a-b020-d879ea607233">

```suggestion
		$parsed_query_id                = $block->attributes['queryId'] ?? null;
		$is_enhanced_pagination_enabled = ! ( $block->attributes['forcePageReload'] ?? false );
```

Reply:
Oh good catch! Looks like a copy/paste fail.